rest-service/manager_rest/constants.py

```diff
@@ -49,3 +49,5 @@
 FILE_SERVER_PLUGINS_FOLDER = 'plugins'
 
 BROKER_SSL_PORT = 5671
+
+SECURITY_FILE_LOCATION = '/opt/manager/rest-security.conf'
```

Comment:
We already have this string in integration tests. Make sure to remove it there, and use the same constant.